test/test_filter_base_diagnostics_timestamps.cpp

```diff
@@ -0,0 +1,421 @@
+/*
+ * Copyright (c) 2014, 2015, 2016, 2018 Charles River Analytics, Inc.
```

Comment:
Not sure that CRA needs to hold the license, if you made the file. Again, not a legal expert. :)

Reply:
Here we have not created new file, we have actually imported this test file from ros1 kinetic-devel and made changes as per Ros2 format.